python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -4275,16 +4275,14 @@ def test_memory_usage_string():
     )
 
     # Check string column
-    assert (
-        gdf.B.memory_usage(deep=True, index=False)
-        == gdf.B._column.nvstrings.device_memory()
+    assert gdf.B.memory_usage(deep=True, index=False) == df.B.memory_usage(
+        deep=True, index=False
     )
 
     # Check string index
-    assert (
-        gdf.set_index("B").index.memory_usage(deep=True)
-        == gdf.B._column.nvstrings.device_memory()
-    )
+    assert gdf.set_index("B").index.memory_usage(
+        deep=True
+    ) == df.B.memory_usage(deep=True, index=False)
```

Comment:
This test is already marked as xfail but this definitely won't work, will need to revisit in the future.